types/concurrently/index.d.ts

```diff
@@ -19,6 +19,13 @@ declare namespace concurrently {
         name?: string;
         prefixColor?: string;
     }
+    interface ExitInfos {
+        command: CommandObj;
+        /** process completion index */
+        index: number;
+        /** process exit code */
+        exitCode: number;
```

Comment:
Can this be `null`, per the concurrently FAQ?

Reply:
From what I've seen, it will not be `null`. 

- `concurrently` will set the `exitCode` to the `signal` if the `exitCode` comes through as `null` [source](https://github.com/kimmobrunfeldt/concurrently/blob/master/src/command.js#L43)

- In the node docs, they say either `exitCode` or `signal` **will always be non-null** on an `exit` event [source](https://nodejs.org/api/child_process.html#child_process_event_exit)

- When the signal is `SIGINT`, `concurrently` sets the `exitCode` to 0: [source](https://github.com/kimmobrunfeldt/concurrently/blob/master/src/flow-control/kill-on-signal.js#L20)

***

The only `signal` exit events I've produced locally have been `SIGINT`, so I've never seen the `exitCode` be anything but a `number`.

I suppose if the `signal` was another `string` besides `SIGINT`, the `exitCode` returned here might be a `string`. What do you think? 

I can try to reproduce this locally in the coming days.

Reply:
I think it's reasonable to leave it as a number. Thanks for the investigative work!